hw_8_new.py

```diff
@@ -0,0 +1,57 @@
+# Написати власний декоратор, задачею якого має бути друк назви функції і часу, коли вона була викликана.
+# Декоратор має працювати для різних функцій однаково.
+import time
+def my_decorator(func):
+    def deco_func():
+        print(time.strftime("%H:%M:%S"))
+        func()
```

Comment:
Assign the result of the func call and return in after the last print to be able to decorate functions that return some value

Reply:
@Vadoomz  Looks like you missed this comment